ynr/apps/candidates/tests/test_models.py

```diff
@@ -43,6 +52,27 @@ def test_get_results_url(self):
         assert other.results_button_text == "Mark elected candidates"
 
 
+class TestBallotMethods(TestCase, SingleBallotStatesMixin):
```

Comment:
There are no actual tests in this class - has something been lost in your rebase perhaps?